tests/com/tony/server/MainTest.java

```diff
@@ -1,43 +1,45 @@
 package com.tony.server;
 
-import com.tony.server.response.GetResponse;
+import com.tony.server.response.FileContentResponse;
 import com.tony.server.response.PutPostResponse;
 import org.junit.Test;
 
 import java.util.ArrayList;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 
 public class MainTest {
     @Test
     public void testCreateURIListHasAllURIFromDirectory() throws Exception {
         String directory = System.getProperty("user.dir") + "/public/";
```

Comment:
This test is pretty fragile in that it depends on certain files actually existing in the public folder. What are some other options here?